config.json

```diff
@@ -147,6 +147,18 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": []
+    },
+    {
+      "slug": "retree",
+      "uuid": "27856806-0dee-409f-8047-357de07b914d",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 3,
```

Comment:
Difficulty should be 1 since all other exercises are. But would be nice to have another PR to add proper difficulty level for all exercises.